### PR TITLE
CHANGED: Moved Navbar to its own React Component and..... Closes #53

### PR DESCRIPTION
**Pull request by efaraz27**
…  ADDed the signin signout button if the user is not logged in

### CODE REVIEW

File: web/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              <Button
+                colorScheme="teal"
+                variant="solid"
+                onClick={() => {
+                  setIsLoggedIn(true);
+                }}
+              >
+                Sign In
+              </Button>
+              <Button colorScheme="teal" variant="outline">
```
Comment (xTrig) [inline web/components/Navbar.js:111]: Maybe let this button redirect users to /signup?

You can use something like:
```suggestion
              <Button colorScheme="teal" variant="outline" onclick={() => Router.push('/signup')}>
```

May have to import the Next.js Router

**Review by xTrig — APPROVED** [review]
Looks good! Nice work!

**Comment by xTrig** [comment]
> … signout button if the user is not logged in

Also could you include: `Closes #50` so that when this is merged, it will automatically close the issue too? That's what we do whenever available.

File: web/components/Navbar.js (hunk 1 above)
Comment (efaraz27) [reply web/components/Navbar.js:111]: sure i'll get it done

**Comment by efaraz27** [comment]
Should close #53

**Review by xTrig — APPROVED** [review]

**Pull request merged**